docs/integrations_api.yml

```diff
@@ -456,86 +333,207 @@ paths:
           $ref: "#/responses/InternalServerError"
 
     delete:
-      summary: Remove firmware image
+      summary: Delete the firmware image
+      description: |
+        Deletes the firmware image from file and images storage.
```

Comment:
We probably should mention that images used in deployment in progress can't be deleted until deployment finishes (they get locked for that time)